tox/__init__.py

```diff
@@ -1,89 +1,34 @@
-import os
-import signal
-
+import pluggy
 from pkg_resources import DistributionNotFound
 from pkg_resources import get_distribution
 
-from .hookspecs import hookimpl
+from . import exception
+from .constants import INFO
+from .constants import PIP
+from .constants import PYTHON
 from .hookspecs import hookspec
 
+__all__ = (
+    'cmdline', 'exception', '__version__',
+    'PYTHON', 'INFO', 'PIP',
+    'hookspec', 'hookimpl',  # DEPRECATED will be removed from API - see warning below
```

Comment:
the way you laid this out here it's not clear for me if both are deprecated or not; also shall we raise warning instead of deprecated; or should that be part of another PR?